tests/test-analysis-module.xml

```diff
@@ -1,32 +1,41 @@
 <dqm4hep>
   
   <!-- Application settings -->
-  <settings>
-    <parameter name="CyclePeriod"> 5 </parameter>
+  <settings mode="EventReader">
+    <parameter name="EnableStatistics"> false </parameter>
+    <parameter name="EventReader"> GenericEventXMLReader </parameter>
+    <parameter name="EventFileName"> /home/$ENV{USER}/soft/dqm4hep-core/tests/genevt_dummy_data.xml </parameter>
```

Comment:
This path won't be valid everywhere (like on travis)

Reply:
Yes sure it was not intended to be valid everywhere. We don't run unit tests with this one